akka-http-core/src/test/scala/akka/http/scaladsl/settings/SettingsEqualitySpec.scala

```diff
@@ -22,13 +22,13 @@ class SettingsEqualitySpec extends AnyWordSpec with Matchers {
       s1.toString should startWith("ConnectionPoolSettings(")
     }
 
-    "hold for ParserSettings" in {
+    "hold for ParserSettings.forServer" in {
       val s1 = ParserSettings(config)
       val s2 = ParserSettings(config)
 
       s1 shouldBe s2
       s1.toString should startWith("ParserSettings(")
-    }
+    }: @silent("apply in object ParserSettings is deprecated")
```

Comment:
Cool trick! ... but shouldn't we actually use `forServer` (or `forClient`) here instead of silencing the warning?

Reply:
Yes, but those only work from an ActorSystem currently.

Reply:
ahh didn't think of that.